debian/control.in

```diff
@@ -0,0 +1,10 @@
+Package: ckan
+Version: @VERSION@
+Architecture: all
+Section: utils
+Priority: optional
+Depends: mono-runtime (>=4.0), ca-certificates-mono, libmono-microsoft-csharp4.0-cil, liblog4net1.2-cil, libnewtonsoft-json5.0-cil
+Maintainer: Michele Ippolito <m.ippolito@outlook.com>
```

Comment:
I'm assuming this is you? I'm looking into whether we can set up an @ksp-ckan.org email address forward

Reply:
No, that's copied forward from the old CKAN-deb repo.
I've changed it to authors@ksp-ckan.org.